animal-sniffer-maven-plugin/src/it/jdk14-with-sig14-from-cli/invoker.properties

```diff
@@ -1 +1 @@
-invoker.goals=org.codehaus.mojo:animal-sniffer-maven-plugin:1.15-SNAPSHOT:check -Danimal.sniffer.signature=org.codehaus.mojo.signature:java14:1.0
```

Comment:
Why was this hard-coding a version? Do not know, but it did not work after the 1.15 release…